src/const/props.ts

```diff
@@ -21,6 +21,31 @@ type FlexboxSelfAlignment = 'self-start' | 'self-end'
 type FlexboxOverflowAlignment = 'safe center' | 'unsafe center'
 
 export interface GenericProps {
+  /**
+   * Specifies direction of flex-items
+   * @css `flex-direction`
+   * @see https://developer.mozilla.org/en-US/docs/Web/CSS/flex-direction
+   */
+  flexDirection?: string
+  /**
+   * Specifies the "flex shrink factor",
```

Comment:
Please end the explanatory comment with a dot :) Thanks.
You can also omit the quotes around flex shrinking factor, I think it's a term.